File: GARI_Umbria_2026-08-03/DotNet/AgronicaCore_2010/AgronicaCoreAnagrafeBIZ/model/Rubrica.vb
Imports System.Runtime.CompilerServices
Imports AgronicaCoreModelsSTD.anagrafiche

Namespace AnagrafeNG

    Public Class Rubrica

        Public Property Rubrica_Cod As Integer
        Public Property Tipologia As String
        Public Property Valore As String
        'public property Descrizione As String

    End Class
End Namespace

Module StringExtensions
    <Extension()>
    Public Function ToRubrica(ByVal rub As RubricaVoci) As AnagrafeNG.Rubrica
        Dim r As New AnagrafeNG.Rubrica()
        r.Rubrica_Cod = rub.rubrica.codice
        r.Tipologia = rub.rubrica.tipologia
        r.Valore = rub.valore
        Return r
    End Function
End Module

File: GARI_Umbria_2026-08-03/DotNet/AgronicaCoreStd/AgronicaCoreModelsSTD/anagrafiche/Rubrica.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace AgronicaCoreModelsSTD.anagrafiche
{
    public class Rubrica
    {

        public int codice { get; set; }
        public string tipologia { get; set; }
        public bool flag_cancellazione { get; set; }

        public Rubrica(int codice)
        {
            this.codice = codice;
            this.flag_cancellazione = false;
        }

        public Rubrica()
        {

        }
    }
}
